lib/info.js

```diff
@@ -9,9 +9,17 @@ const getInfo = async (dhpkgid) => {
   const { publisher, pkg } = utils.parseIdentifier(dhpkgid)
 
   const metadata = await getMetadata(publisher, pkg)
-  const readme = await getReadme(urljoin(metadata.bitstore_url, 'README.md'))
+  let readme = await getReadme(urljoin(metadata.bitstore_url, 'README.md'))
```

Comment:
`README.md` might now always be the case, we accept README and README.txt as well. We need to think more about this